### PR TITLE
class PaymentType instead of interface

### PR DESCRIPTION
**Pull request by KaiNissen**
follow-up to #175

### CODE REVIEW

File: src/Domain/Model/PaymentType.php [hunk 1]
```diff
@@ -17,4 +18,8 @@
 	const CASH = 'CASH';
 	const BITCOIN = 'BTC';
 
+	public static function getPaymentTypes() {
+		return ( new \ReflectionClass( self::class ) )->getConstants();
```
Comment (JeroenDeDauw) [inline src/Domain/Model/PaymentType.php]: reflection

An actually non-evil looking usage of Reflection. Now I've seen it all.

Typical PHP... have `get_class_vars` and `get_class_methods`, but no `get_class_constants`

File: src/FunFunFactory.php [hunk 2]
```diff
@@ -195,16 +194,6 @@ private function newPimple(): \Pimple {
 			] );
 		} );
 
-		// In the future, this could be locale-specific or filled from a DB table
-		$pimple['payment_types'] = $pimple->share( function() {
-			return new PaymentTypes( [
```
Comment (JeroenDeDauw) [inline src/FunFunFactory.php]: Is PaymentTypes not unused now?

File: src/Domain/Model/PaymentType.php [hunk 3]
```diff
@@ -17,4 +18,8 @@
 	const CASH = 'CASH';
 	const BITCOIN = 'BTC';
 
+	public static function getPaymentTypes() {
```
Comment (JeroenDeDauw) [inline src/Domain/Model/PaymentType.php]: : array

File: src/FunFunFactory.php (hunk 2 above)
Comment (KaiNissen) [inline src/FunFunFactory.php]: Yes, did that on a different branch, rebased now.

**Pull request merged**